commit 4ae7f6320aeb599c10b057abe7fcfcc277532959
Author: Kees Cook <kees@kernel.org>
Date:   Mon Sep 25 10:30:58 2023 -0700

    MAINTAINERS: hardening: Add Gustavo as Reviewer
    
    It's an oversight to not have already listed Gustavo here. Add him as a
    Reviewer.
    
    Cc: Gustavo A. R. Silva <gustavoars@kernel.org>
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 741285b8246e..5f18ed0fbd42 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11393,6 +11393,7 @@ F:	usr/
 
 KERNEL HARDENING (not covered by other areas)
 M:	Kees Cook <keescook@chromium.org>
+R:	Gustavo A. R. Silva <gustavoars@kernel.org>
 L:	linux-hardening@vger.kernel.org
 S:	Supported
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/kees/linux.git for-next/hardening